src/BadasoLMSModule.php

```diff
@@ -2,12 +2,12 @@
 
 namespace Uasoft\Badaso\Module\LMS;
 
-class BadasoLMSModule {
```

Comment:
do not commit this again, please check previously existing file on the staging branch